dll/win32/imageres/imageres.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * PROJECT:     ReactOS imageres.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     imageres.dll
+ * COPYRIGHT:   Copyright 2023 Ethan Rodensky <splitwirez@gmail.com>
+ */
+
+#ifndef _IMAGERES_H_
+#define _IMAGERES_H_
```

Comment:
#pragma once instead? Should be shorter and faster at compile-time.

Reply:
Huh, TIL that's a thing you can do. Thanks, just took care of that :+1: